lib/Runtime/PlatformAgnostic/Platform/Common/Intl.cpp

```diff
@@ -467,14 +491,20 @@ namespace Intl
         return ret;
     }
 
-    bool IsLocaleAvailable(_In_z_ const char *locale)
+    bool IsLocaleAvailable(_In_z_ const char *langtag)
```

Comment:
>Locale [](start = 11, length = 6)

change function name to either langtag or localeID (or note spec refs locale == LocaleID)